test/sql/CMakeLists.txt

```diff
@@ -74,8 +74,11 @@ set(SOLO_TESTS
   alter
   alternate_users
   bgw_launcher
-  chunk_utils.sql
-  index
+  chunk_utils
+  index-11
```

Comment:
Could you clarify why we need to add this to SOLO_TESTS as well? We already have this in the templated list (line 68).

Reply:
SOLO_TESTS doesnt actually run any tests it's just a marker that those tests will not run in parallel with other tests but it still needs to be in one of the other lists to actually run

Reply:
Well we had this before for `index` test. Since index test made as a template, I thought it is necessary to update it here as well. I am not aware why index test was put in the SOLO tests list in the first place, likely it has something to do with tablespace